src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.tsx

```diff
@@ -28,6 +35,39 @@ export const ActionSelectionForm = ({
 }: ActionSelectionFormProps) => {
   const canManage = canSurrender;
 
+  const { setOverlayContent, setOverlayVisible } = useContext(OverlayContext);
+  const onNoAccessHandle = () => {
+    setOverlayContent(
+      <DocumentTransferMessage title="No manage assets access" isTitleIconSuccess={false}>
```

Comment:
Possible to export different types of messages and then let the overlay go figure out the content? 

The interface will be like

```
showOverlay(Messages.NO_MANAGE_ACCESS)
```

then those with extra props like {address} will be:

```
showOverlay(Messages.SUCCESS_BENEFICIARY_CHANGE, {address: 0x1234""})
```

With this approach it's easier to read what is done in this component instead of trying to figure out the internal workings of the overlay.

Reply:
updated, but not very sure if i implemented correctly